src/components/SearchBar.vue

```diff
@@ -36,7 +32,12 @@
             onResultSelect(event) {
                 this.map.indoors.setFloor(event.result.data.floor_id);
                 this.map.setView(event.result.location.latLng, 20);
-                this.route({map: this.map, destination: [event.result.data.lon, event.result.data.lat, event.result.data.floor_id]});
+                this.$store.dispatch('route', {
+                    map: this.map,
+                    destination: [event.result.data.lon, event.result.data.lat, event.result.data.floor_id]
+                }).then((response) => {
+                    console.log(response);
+                })
```

Comment:
Making use of the response returned by dispatch in the event that any routing fails

Reply:
why are we not using mapactions here?

Reply:
Using "this.$store.dispatch" is an alternative to mapActions. (Since the store is injected through the entire application including its children, the store can be accessed by this.$store.[]) I can change it to using mapAction if you would prefer
![image](https://user-images.githubusercontent.com/24628243/76153543-4c07d300-609b-11ea-97e5-6607b89c201b.png)